CMake/resolve_dependency_modules/folly/CMakeLists.txt

```diff
@@ -14,9 +14,9 @@
 project(Folly)
 cmake_minimum_required(VERSION 3.14)
 
-set(VELOX_FOLLY_BUILD_VERSION v2023.12.04.00)
+set(VELOX_FOLLY_BUILD_VERSION v2024.02.26.00)
```

Comment:
This will only set the folly version when you bundle it - if we need to enforce changes everywhere we should change the scripts in scripts/setup-*.sh too.

Reply:
Good point. Just changed in the scripts I could find as well. 

We should find a way to have this defined in a single place though :)

Reply:
Folly doesn't provide a version file right? So we can't add it to resolve_dependency like for the other deps to enforce the version?